imap_processing/spice/time.py

```diff
@@ -13,6 +13,30 @@
 
 TICK_DURATION = 2e-5  # 20 microseconds as defined in imap_sclk_0000.tsc
 
+# Hard code the J2000 epoch. This allows for CDF epoch to be converted without
+# use of SPICE though it should be noted that this results in a 5-second error
+# due to the occurrence of 5 leap-seconds since the J2000 epoch.
```

Comment:
Arguably a **TODO** should go here, since this is where the epoch is defined, and where the 5 leapsecond fix will go, right?

Reply:
True Nat. I meant to ask Tim if there is a ticket to revisit for this?

Reply:
I will add the TODO. Yes, there is a ticket for this. There are two ways to fix this each with a pro and con. See ticket: #1208

To clarify, this J2000EPOCH value is correct. The bug is in the code in the `cdf/utils.py` here:
```
  dt64 = J2000_EPOCH + dataset["epoch"].values[0].astype("timedelta64[ns]")
  start_time = np.datetime_as_string(dt64, unit="D").replace("-", "")
```